source/deployment/ldap-group-constrained-team-channel.rst

```diff
@@ -0,0 +1,78 @@
+.. _ldap-group-constrained-team-channel:
+
+Using AD/LDAP Synchronized Groups to Manage Team or Private Channel Membership
+===============================================================
+Mattermost groups created with `synchronized AD/LDAP groups <https://docs.mattermost.com/deployment/ldap-group-sync.html>`_ can be used to manage the membership of private teams and private channels. When a team or private channel is managed by synchronized groups, users will be added and removed based on their membership to the synchronized AD/LDAP group. 
+
+For instance, you may have a AD/LDAP group that contains your development team that you want to synchronize to a developer team.  By using this feature, new developers will get added to the team when they are added to the synchronized AD/LDAP group and they will be removed from the team when removed from the AD/LDAP group. 
+
+Similarly, you may have a AD/LDAP group that contains your leadership team that you want to synchronize to a private channel for coordination and updates.  This feature will help control the membership of the channel so that users outside of the synchronized group are prevented from being added the channel mistakenly. 
+
+On teams that are managed by synchronized groups, users outside of the group are restricted from:
+
+ - invitation through a team invite link 
+ - invitations through an email invite 
+ 
+Similarily on private channels that are managed by synchronized groups, users outside of the group are restricted from:
+
+ - invitation through a mention
+ - invitation through the /invite slash command 
+ - directly added to the channel with “add members”
+
+Users, however, can remove themselves from teams and private channels managed by synchronized groups.  
+
+Managing Membership of a Team or Channel with Synchronized Groups
+----------
+
+To manage membership of a private team with synchronized groups: 
+
+1. Ensure there is at least one group already associated to the team. You can view and add default teams to a group via **System Console > User Management > Groups > Group Configuration**. Please see more information on adding default teams and channels `here <https://docs.mattermost.com/deployment/ldap-group-sync.html#add-default-teams-or-channels-for-the-group>`_. Additionally, you can use the CLI tool to view if there is already a group associated to the team by running the `group team list CLI command <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-group-team-list>`_. 
+2. Ensure **Team Settings > General > Allow any user with an account on this server to join this team** is set to ``No``. 
```

Comment:
This step won't do any harm but it is not a requirement. `$ mattermost group team enable [team]` will work and the value of `AllowOpenInvite` is ignored.

Reply:
It will be a requirement in the UI, so I will keep it at this time.  Thanks for clarifying.